modules/core/src/main/scala/org/scalasteward/core/git/FileGitAlg.scala

```diff
@@ -47,6 +47,11 @@ final class FileGitAlg[F[_]](config: GitCfg)(implicit
   override def checkoutBranch(repo: File, branch: Branch): F[Unit] =
     git_("checkout", branch.name)(repo).void
 
+  override def checkIgnore(repo: File, file: String): F[Boolean] =
+    git_("check-ignore", file)(repo)
+      .as(true)
+      .recover { case _: ProcessFailedException => false }
```

Comment:
I suggest to only recover here if the process [exits with 1](https://git-scm.com/docs/git-check-ignore#_exit_status):
```scala
.recover { case ex: ProcessFailedException if ex.exitValue === 1 => false }
```
This would require to make `exitValue` a field of `ProcessFailedException`. It is currently only a constructor parameter.

Reply:
Updated

Reply:
Thanks!